README.md

```diff
@@ -588,7 +588,7 @@ Other Style Guides
     }
 
     // good
-    const foo = function bar() {
+    const foo = function foo () {
```

Comment:
No prose - this example constitutes the preference. There's actually an eslint rule that enforces that they be different, but I'm not planning on enabling it in the config by default.